pkg/lib/machine-info.js

```diff
@@ -193,3 +193,76 @@ export function udev_info(address) {
     }
     return pr;
 }
+
+const memoryRE = /^([ \w]+): (.*)/;
+
+// Process the dmidecode output and create a mapping of locator to DIMM properties
+function parseMemoryInfo(text) {
+    var info = {};
+    text.split("\n\n").map(paragraph => {
+        let locator = null;
+        let props = {};
+        paragraph = paragraph.trim();
+        if (!paragraph) return;
```

Comment:
Please break the line after if() conditions. Here and in some lines  below as well.

Reply:
Made changes now